CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [Log4Net_v1.0.2]
```

Comment:
Is this a Bugfix release since this issue was labeled a bug?  if so, we add the same format as:
### Bugfix release

Reply:
Done.  I also added the date of the release for consistency as well.  You'll need to re-review, please.